acceptance/tests/base/dsl/helpers/host_helpers/rsync_to_test.rb

```diff
@@ -12,8 +11,9 @@ def rsync_to_with_backups hosts, local_filename, remote_dir
         ) # return of block is whether or not we're done repeating
         return result.success? if result.is_a? Rsync::Result
 
-        result.each do |individual_result| 
+        result.each do |individual_result|
```

Comment:
Or we clean it up as https://github.com/voxpupuli/beaker/pull/1776.

Reply:
done